pages/searchResults.page.js

```diff
@@ -0,0 +1,51 @@
+let BasePage = require('./base.page');
+let TextBox = require('../elements/textBox');
+let WebButton = require('../elements/button');
+
+let searchResultsFirstLinkLocator = by.css('.products-list > li:nth-child(1) > div.item-info > p > a');
+let itemPageHeaderLocator = by.css('div.heading > h1');
+
+class SearchResultsPage extends BasePage {
+    async waitForSearchPageToBeAvailable() {
+        await this.getPageHeaderElement().waitForElementToBeVisible(10000);
+    }
+
+    // action methods
+    async getSearchResults() {
```

Comment:
make this this method just to return the count of search results. No need to do any `expect` here, expectation should be only in test body.

Reply:
done